app/src/main/java/com/example/songme/mediaplayer/MediaPlayerContract.kt

```diff
@@ -12,9 +12,10 @@ interface MediaPlayerContract {
     fun nextTrack()
     fun prevTrack()
     fun changeLoop()
-    fun changeShuffled()
+    fun changeShuffled(isShuffle: Boolean)
```

Comment:
luồng điều khiển của MediaPlayerManager bị thay đổi dựa vào tham số truyền vào -> Nó bị điều khiển bởi thằng khác 
tách ra làm 2 hàm riêng 
```suggestion
    fun turnOnShuffledMode()
    fun turnOffShuffleMode()
```

Reply:
nếu như vậy, ví dụ trong trường hợp tham số đầu vào có 7 trường hợp khác nhau thì mình cũng tách thành 7 hàm sao. Nếu làm ntn thì mục đích của nó còn làm đc gì lớn hơn việc tiết kiệm đc một số vòng lặp k ?